cider/exec/template/IRCodegen.cpp

```diff
@@ -184,6 +184,18 @@ std::unique_ptr<CodegenColValues> CodeGenerator::codegen(const Analyzer::Expr* e
   if (constant) {
     return codegenConstantExpr(constant, co);
   }
+  auto dateadd_expr = dynamic_cast<const Analyzer::DateaddExpr*>(expr);
+  if (dateadd_expr) {
+    return {codegenDateAdd(dateadd_expr, co)};
```

Comment:
why use a ```{}``` covered the codegenDateAdd(dateadd_expr, co) instead of just codegenDateAdd(dateadd_expr, co)?

Reply:
I didn't notice that，i'll remove```{}```.